packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -391,11 +391,30 @@ export type SdkModelPropertyType =
   | SdkBodyParameter
   | SdkHeaderParameter;
 
+export interface MultipartOptionsType {
+  // whether name is defined in Typespec. For multipart/mixed, name may not be defined for some parts
+  isNameDefined: boolean;
+  // whether this part is for file
+  isFilePart: boolean;
+  // whether this part is multi in request payload
+  multi: boolean;
```

Comment:
full name: multiple or allowMultiple?

Reply:
With its meaning, it equals to "multiple"